src/Movie-form.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import './Movie-form.css'
+
+class MovieForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      unMatchedResult: false
```

Comment:
Good use of a state property in order to check when we want to insert our error message